instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb

```diff
@@ -16,6 +16,9 @@ def dispatch(name, request, response)
               rack_span.name = "#{self.class.name}##{name}" if rack_span.context.valid? && !request.env['action_dispatch.exception']
```

Comment:
Should the following 2 statements also be covered by the conditional above?

Reply:
@fbogsany that makes sense and I will adjust to use `rack_span.recording?`

As I understand it, `rack_span.name` defaults to the request path. Should this be set to use the filtered path if the exception is present?

Reply:
`rack_span.name` will default to the request path, but in general, when an exception occurs, the `rack_span.name` will already have been updated to `<SomeClassName>#<some action>`. We want to avoid updating it again to `ExceptionController#<some action>` when an exception is present. I am not sure if there's a corner case where an exception occurs before the request is routed to a controller+action.

If the corner case exists (I'd want to see evidence of that first), then we might want to change the behaviour. It becomes a little challenging because `span.name` is not supposed to be readable (and isn't, with the API `Span`), so we can't simply parse it as a URL.